docs/releases/2.2.1.txt

```diff
@@ -18,7 +18,10 @@ Bugfixes
 * Fixed a regression in Django 2.2 that caused a crash when loading the
   template for the technical 500 debug page (:ticket:`30324`).
 
+* Fixed a regression in Django 2.2 that caused a crash when IntegerField
+  limits are callable.
```

Comment:
Add `` (:ticket:`30328`)`` to the end before the `.` and move this entry to the end.